src/app.ts

```diff
@@ -87,29 +87,35 @@ app.delete('/boards/:board_id', Board.delete)
 app.post('/boards', Board.create)
 
 // ---------    Card Routes   ---------
-app.get('/cards/:card_id', Card.getOneById)
-app.get('/boards/:board_id/lists/:list_id/cards', Card.getAllFromListId)
-app.put('/cards', Card.update)
-app.delete('/cards/:card_id', Card.delete)
-app.post('/boards/:board_id/lists/:list_id/cards', Card.create)
+app.get('/cards/:id', Card.getOneById)  
```

Comment:
I think it is better to group the routes by entity instead of verb.

Reply:
Done